---
title: "Playing the Game"
book: "players-handbook"
slug: "players-handbook/playing-the-game"
weight: 102
type: rules
draft: false
---
Playing the GUTS+ System requires only the occasional [D6]({{< relref "definition/nDs.md" >}}) roll and a bit of imagination.

<h1 class="is-inline" id="the-basics">The Basics</h1>[<span class="icon"><i class="fa fa-link"></i></span>](#the-basics)

Role playing is a bit like collaborating with a group to tell a story with special rules in place that prevent the story from going too far off the rails. The [GM]({{< relref "definition/game-master.md" >}}) sets up the world and the situation, and it's up to you and your fellow players to work through that situation together using your characters.

Oftentimes, the GM will start a play session by describing a scene and some kind of circumstance that would bring your characters together for an adventure. It's up to you as the players to ask questions both in character ([IC]({{< relref "definition/in-character.md" >}})) and out of character ([OOC]({{< relref "definition/out-of-character.md" >}})) and fill in the blanks so you can be as successful as possible when moving forward.

When your character asks questions and tries to perform certain actions, the GM will sometimes ask you to make a success check. Depending on what you are trying to roll success for, your stats may influence the outcome, raising your roll total by the value of the stat—the GM should tell you which stat to use if you don't know.

You will occasionally roll for success _against_ a non-playable character ([NPC]({{< relref "definition/non-playable-character.md" >}})). The character being rolled against is the "defender." Both you and the DM will roll, and whoever's roll is higher wins. If you both roll the same number, then the person who is trying to succeed wins the roll.

As you continue playing, your characters will grow stronger, and the game will get harder. Just keep moving forward, and you'll win eventually!

<h1 class="is-inline" id="handling-conflict">Handling Conflict</h1>[<span class="icon"><i class="fa fa-link"></i></span>](#handling-conflict)

Conflict is the cornerstone of plot. The game you are playing will likely have a lot of different conflicts, often happening at the same time, as well as a lot of different kinds of conflicts. Whether you are trying to find someone's lost cat or trying to work against a dictator, there are many different ways to go about handling conflict.

Conflicts are handled using a combination of die rolls and logical appeals made to the GM. When an appeal is not accepted by the GM, you must roll a number D6s to determine your success. See the [Success Scale](#success-scale) to understand how success is determined. When you begin, you will only be rolling 1 D6, but as your character grows stronger, you will need to roll more to keep up with the rising difficulty level.

Many role-playing games make heavy use of combat to overcome conflicts, but it might not always be the best option. Try different things to see what the best outcome might be. The GM should be flexible enough to handle whatever you throw at them and respond accordingly. If you come across someone acting strangely, instead of trying to beat the information out of them, maybe try to ask them what's wrong.

Sometimes, though, [Combat](#combat) really is the only option.

### Success Scale

How successful your character is at performing a certain action is determined by what number you roll on a [D6]({{< relref "definition/nDs.md" >}}) die. Occasionally, the DM will have you roll for a particular stat. In these cases, roll 1D6 plus an additional D6 per 3 stat points your character has—the number of dice you roll increases the range factor for each success (or failure) category.

There are 2 types of success rolls: checks and contests.

#### Check

Check rolls are what determine your success at performing actions that do not involve other (unwilling) living creatures. The scale for success is:

```
1. Negative impact
2. Failure
3. Near success (GM decides)
4. Full success
5. Good success
6. Positive impact
```

As mentioned above, the scale changes for each additional die you roll. For example, if your character has 3 stat points in Gumption, roll 2D6 and scale the numbers up: 1–2 = 1, 3–4 = 2, 5–6 = 3, 7–8 = 4, 9–10 = 5, 11–12 = 6. Notice that with 2D6 it is impossible to roll 1, which means that as your character grows more experienced, they are less and less likely to make a roll that results in a negative impact.

#### Contest

If you need to perform an action against an unwilling member, you will need to roll a contest against them. Both you and your opponent roll the relevant number of dice, and you subtract their die roll from yours. The success scale is as follows:

```
-2. Negative impact
-1. Failure
0. Near success (GM decides)
1. Full success
2. Good success
3. Positive impact
```

Anything above or below these values counts as the extreme end. Scaling Contest values depends on whether you get a positive or negative value. Scale the success table based on the number of dice your _opponent_ rolls for _positive values_ and the number of dice _you_ roll for _negative_ values.

For example, if you rolls 1D6 and you get a 6 and your opponent rolls 2D6 and they get a 4, the contest value is 2, but the scale value is only 1 because your opponent rolled two D6's. On the other side, if you had rolled 2D6 and got 4 and your opponent got a 6 from whatever they rolled, the contest value is -2, but the scale value is only -1 because you rolled two D6's.

#### "Near Success"

Whenever you roll a 3 check or a 0 contest, the result is at the discretion of the GM. They may take the opportunity to advance the situation somehow or they may decide that you need to roll better next time based on the context. Feel free to plead your case with the GM, though!

<h1 class="is-inline" id="combat">Combat</h1>[<span class="icon"><i class="fa fa-link"></i></span>](#combat)

If you do choose to handle your conflicts with violence, combat is a fairly straightforward affair. Characters take turns choosing who and how they want to attack in the order of highest to lowest Gumption stat. If there's a tie in this stat, the GM will decide how the turn order will fall.

When you attack someone or something in the game, you need to declare what you are trying to do and then make a [contest roll](#contest) against your opponent. Depending on what you are trying to do, you will be using certain stats to determine how many dice to roll and how to augment your roll. The GM will tell you the result of your attack unless you roll well enough to make the decision yourself. Once your attack is done, the next person in the list makes their move.

Note that you can also choose to take any other action aside from attacking if you wish. Maybe you can defuse the situation instead! Or perhaps you have the ability to heal a partner's injuries—that's also something you can do on your turn!

### Timing

Time spent differs from turn to turn depending on what action is done, but in general, most turns take about 5 seconds to complete in-game. Also, you can specify _when_ your turn happens in relation to other turns that have already been taken, for example "at the same time as" another character is attacking one monster, you attack another, or right after a player knocked an enemy's shield away, you take advantage of the opening in their defense.

### Holding Your Turn

Sometimes, you will want to wait to perform an action until something else happens, for example, to perform a group attack or a counterattack or some other complex maneuver. You may hold your turn at any time, but if you do not use your held turn before your next turn comes around, you will have lost that turn.

### Weapons

In order to use a weapon, your character must be holding it in their Hand (i.e. not in a Bag), and they must have a place to put anything that they might already be holdingninstead of their weapon. Beyond this, the effect of the weapon is governed by logic and the Success Scale. For example, using bare fists may not be very effective against metal unless you've got a really good reason as to why it actually would be, but a knife might be able to cut through some wood.

### Taking Damage

When you are attacked and take damage, your character will receive injuries depending on where they were hit. See [Health](#health) to learn about the health and injuries systems in GUTS+.

### Completing Combat

Combat goes until the GM declares that the opponent has been defeated. This could be by knocking them out or making them run or otherwise convincing them to stop fighting.

<h1 class="is-inline" id="health">Health</h1>[<span class="icon"><i class="fa fa-link"></i></span>](#health)

Most role-playing game systems utilize a "hit points" (HP) system, but in GUTS+, a character's health is determined by the number of injuries sustained to various parts of their body and the strain placed upon their mind and body.

### Injuries

The following chart shows the areas of the body that can be injured and how many times an injury can be sustained before it becomes fatal:

<figure class="image is-3by4" style="max-width:400px;">
  <img src="/images/injury-chart.svg" alt="Injury Chart" />
</figure>

Whenever you take an injury, mark it on your character sheet. If a part of the body sustains the maximum number of Minor injuries that it can have, the next Minor injury instantly becomes a Major injury, and if the maximum number of Major injuries is taken, use of that body part becomes limited and will affect what your character can do. If you receive any Major injuries beyond the maximum number, you lose use of that body part completely. Losing use of either of your character's Hands reduces your [inventory space](#inventory), and "losing the use of" your character's Head means your character immediately becomes unconscious.

If your character receives _more than_ 6 Major (non-Head) injuries (i.e. 7 or more), they will fall unconscious. If they are not recovered before enough time passes, they will die, so be careful!

### Strain

In addition to Injuries, performing a taxing action for a sustained period of time can add Strain to your character. If your character receives more than 10 points of Strain before they are able to remove any Strain, they can either receive injuries or be knocked unconscious from exertion.

Any additional strain received from performing actions inflicts a Minor injury to the body part that is performing it. If you are using [magic]({{< relref "rules/grimoire.md" >}}), strain received inflicts injuries to the Head.

### Recovery

Recovering from an injury is a relatively slow process if not deliberately focused on. Untreated, A minor injury will heal at the rate of 1 injury per in-game day—you choose what injury you wish to be healed—while a major injury will heal at the rate of 1 injury per 5 in-game days.

If you wish to focus on allowing your injuries to heal, resting will allow your injuries to heal twice as fast: 1 minor injury per 1/2 day and 1 major injury per 2 1/2 days. If you or someone in your group has any sort of medical stats, then they can help treat your wound to speed up recovery even more. Some treatments will be instantaneous while others might speed it up to just a couple of hours for recovery. Eating food can also help heal injuries.

Strain is slightly different: once the source of strain is removed, it begins going down immediately at a rate of approximately 1 point per minute. Depending on the type of strain and how soon you resume the stressful task, your strain might be removed all at once or after every couple of turns if in [Combat](#combat).